CODEOWNERS

```
@@ -1,6 +1,6 @@
 * @jeff-mccoy @cmwylie19 @btlghrants @schaeferka
 
 # Additional privileged files
-/CODEOWNERS @jeff-mccoy @austenbryan
-/cosign.pub @jeff-mccoy @austenbryan
-/LICENSE @jeff-mccoy @austenbryan
+/CODEOWNERS @jeff-mccoy @daveworth
+/cosign.pub @jeff-mccoy @daveworth
+/LICENSE @jeff-mccoy @daveworth
```

```suggestion
/LICENSE @jeff-mccoy @austenbryan 
```

Updated on behalf of @austenbryan via Slack DM:
```
Dave as code owner and coding, me as license. He doesn’t need to be on that since Jeff and I will be.
```